### ISSUE TITLE
Size threshold filter 

### ISSUE THREAD

**Issue by fmaussion** [GLIMS-RGI/rgi7_scripts#67]
Because this is how RGI6 did it, I've implemented the size threshold filter to filter *after* rounding, see:

https://github.com/GLIMS-RGI/rgi7_scripts/blob/11816633256f4af1ed24674bad65f296268feebe/workflow/utils.py#L164-L168

The idea behind this was:
- that's how RGI6 did it, and so many outlines in RGI6 are actually slightly smaller than 0.01 km2. Therefore RGI7 in some regions would have less outlines despite of being unchanged to RGI6.
- there is uncertainties in the outlines and therefore the area, and this would take this into account.

But now I regret this decision, and it is still time to change it. 

Thoughts? @ezwelty

**Comment by fmaussion** [comment]
+ the bug that @ezwelty pointed out:

Actually, if I understand this line in correct_geoms from rgi_scripts, you are ignoring secondary polygons smaller than 1e6m2 (1 km2), although this is inconsistent with the print statement (0.1 km2).

    elif np.isclose(area_new, area_bef, atol=1e6):
        print('Smaller than 0.1 km2. Ignoring.')

https://github.com/GLIMS-RGI/rgi7_scripts/blob/11816633256f4af1ed24674bad65f296268feebe/workflow/utils.py#LL348-L349C59

**Comment by ezwelty** [comment]
@fmaussion I think we're talking about two different things. On one hand, the filtering of outlines coming from GLIMS for inclusion in RGI (which I guess is performed by `size_filter`), and on the other whether to keep secondary polygons that result from geometric manipulations on the outlines (performed by `correct_geoms` and in a different flavor in my algorithm for resolving self-overlaps: https://github.com/ezwelty/rgi_links/blob/23c095e12b7fb32f0ab2361d1ea76cc314706c7b/helpers.py#L240).

If you want all of these to be consistent, then why not use `area > 9500` m2, which exactly matches the RGI6 filter and is easier to communicate.

**Comment by fmaussion** [comment]
yes agreed - I'm just putting into question whether `area > 1000 m2` wouldn't be even easier to communicate. Only drawback is that it would be different than RGI6
